src/instant/InstantView.tsx

```diff
@@ -1,9 +1,141 @@
+import { BigNumber } from 'bignumber.js';
+import classnames from 'classnames';
 import * as React from 'react';
+import { createNumberMask } from 'text-mask-addons/dist/textMaskAddons';
+import { theAppContext } from '../AppContext';
+import { BigNumberInput } from '../utils/bigNumberInput/BigNumberInput';
+import { formatPrice } from '../utils/formatters/format';
+import { Button } from '../utils/forms/Buttons';
+import { SettingsIcon, SwapArrows } from '../utils/icons/Icons';
+import * as panelStyling from '../utils/panel/Panel.scss';
+import { TopRightCorner } from '../utils/panel/TopRightCorner';
+import { Asset } from './asset/Asset';
+import { FormChangeKind, InstantFormState, ManualChange, Message, MessageKind } from './instant';
+import * as styles from './Instant.scss';
 
-export class InstantView extends React.Component {
+class TradingSide extends React.Component<any> {
   public render() {
-    return <h1>Instant</h1>;
+    return (
+      <div className={styles.assetPicker}>
+        <Asset currency={this.props.asset} balance={this.props.balance}/>
+        <BigNumberInput
+          type="text"
+          className={styles.input}
+          mask={createNumberMask({
+            allowDecimal: true,
+            decimalLimit: 5,
```

Comment:
Should be parametrized (like `OfferMakeForm`)

Reply:
Actually I've literally copy/pasted it from there so frankly I have no idea by parameterized in here. Please elaborate

Reply:
Tokens have different decimalLimits.
Hardcoded version works, but causes some input problems like ED-157

Reply:
Yeah , just found it. Can we agree like this. I will put a TODO there and once I figure out how to build the list of all assets ( the token selector ) then I will parameterized it .. How does that sound to you ?

Reply:
Ok

Reply:
I've pushed with the TODO in mind and removed the comment so can you recheck and approve :) Thanks